src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,1944 @@
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Called when client gets the all recognition results from vc-daemon.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control</feature>
+        /// <feature>http://tizen.org/feature/microphone</feature>
+        /// <pre>
+        /// The State should be Initialized
+        /// </pre>
+        public static event EventHandler<AllRecognitionResultEventArgs> AllRecognitionResult
+        {
+            add
+            {
+                _allResultDelegate = (RecognizedResult result, IntPtr vcCmdList, IntPtr recognizedText, IntPtr msg, IntPtr userData) =>
+                {
+                    string recognizedString = Marshal.PtrToStringAnsi(recognizedText);
+                    string msgString = Marshal.PtrToStringAnsi(msg);
+                    AllRecognitionResultEventArgs args = new AllRecognitionResultEventArgs(result, recognizedString, msgString);
+                    _allRecognitionResult?.Invoke(null, args);
+                    return true;
+                };
+                ErrorCode error = VcMgrSetAllResultCb(_allResultDelegate, IntPtr.Zero);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Add AllRecognitionResult Failed with error " + error);
+                }
+                else
+                {
+                    _allRecognitionResult += value;
+                }
+            }
+
+            remove
+            {
+                ErrorCode error = VcMgrUnsetAllResultCb();
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Remove SetAllResult Failed with error " + error);
+                }
+
+                _allRecognitionResult -= value;
+                _allResultDelegate = null;
+            }
+        }
+
+        /// <summary>
+        /// Called when client gets the pre recognition results(partial ASR) from vc-daemon.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control</feature>
+        /// <feature>http://tizen.org/feature/microphone</feature>
+        /// <pre>
+        /// The State should be Initialized
+        /// </pre>
+        public static event EventHandler<PreRecognitionResultUpdatedEventArgs> PreRecognitionResultUpdated
+        {
+            add
+            {
+                _preResultDelegate = (PreResultEventType result, IntPtr recognizedText, IntPtr userData) =>
```

Comment:
You need to understand of multiple handler
it is not working below code
``` c#
VoiceControlManagerClient.PreRecognitionResultUpdated += firsthandler;
VoiceControlManagerClient.PreRecognitionResultUpdated += secondhandler;

 VoiceControlManagerClient.PreRecognitionResultUpdated -= firsthandler;

// secondhandler not working



```

Reply:
Please refer this PR
https://github.com/Samsung/TizenFX/pull/812

Reply:
fixed.